homeassistant/components/myuplink/config_flow.py

```diff
@@ -62,5 +72,10 @@ async def async_oauth_create_entry(self, data: dict) -> ConfigFlowResult:
             return self.async_update_reload_and_abort(
                 self._get_reauth_entry(), data=data
             )
+        if self.source == SOURCE_RECONFIGURE:
+            self._abort_if_unique_id_mismatch(reason="account_mismatch")
+            return self.async_update_reload_and_abort(
+                self._get_reconfigure_entry(), data=data
+            )
```

Comment:
This can be combined with the reauth one. I think other integrations have this as well, but I am not sure what a nice way to get the entry in a nice way

Reply:
I followed the guide-lines here https://developers.home-assistant.io/blog/2024/10/21/reauth-reconfigure-helpers At least I think so?